src/components/WorkOrders/MobileWorkingCloseWorkOrderForm.tsx

```diff
@@ -134,37 +139,40 @@ const MobileWorkingCloseWorkOrderForm = ({
           )}
         </div>
 
-        <div
-          style={{
-            display: currentPage === PAGES.FOLLOW_ON_DETAILS ? 'block' : 'none',
-          }}
-        >
-          <h1 className="lbh-heading-h2">Details of further work required</h1>
+        {currentPage === PAGES.FOLLOW_ON_DETAILS && (
+          <div>
+            <h1 className="lbh-heading-h2">Details of further work required</h1>
 
```

Comment:
Hiding the fields rather than removing them caused the new radio field to validate when it shouldnt. Instead, the value is now re-populated via default checked